src/ethRegistrar.ts

```diff
@@ -91,46 +93,20 @@ export function handleNameRenewed(event: NameRenewedEvent): void {
 
 export function handleNameTransferred(event: TransferEvent): void {
   let label = uint256ToByteArray(event.params.tokenId)
-  let registration = new Registration(label.toHex())
```

Comment:
With this change, the registration's `registrant` field is no longer updated when a name is transferred.

Reply:
Changed to save only if a valid registration exits